docs/modules/data-structures/pages/vector-search-overview.adoc

```diff
@@ -43,17 +43,26 @@ The index is based on the link:https://github.com/jbellis/jvector[JVector] libra
 
 Each collection is partitioned and replicated based on the system's general partitioning rules. Data partitioning is carried out using the collection key.
 
-For further information on Hazelcast partitioning, see xref:architecture:data-partitioning.adoc[Data Partitioning and Replication].
+Vector collection is AP data structure and implements standard xref:data-structures:backing-up-maps.adoc#in-memory-backup-types[in-memory backup types].
+Vector collection does not currently support xref:data-structures:backing-up-maps.adoc#enabling-in-memory-backup-reads-embedded-mode [reading from backup] and xref:data-structures:backing-up-maps.adoc#file-based-backups[file based backups].
 
-NOTE: Version 5.5/beta supports partitioning and migration but does not include support for the backup process.
+For further information on Hazelcast partitioning, see xref:architecture:data-partitioning.adoc[Data Partitioning and Replication].
 
 === Data store
+
 Hazelcast stores data in-memory (RAM) for faster access. Presently, the only available data storage option is the JVM heap store.
 
 === Fault Tolerance
+
 Hazelcast distributes storage data across all cluster members.
 In the event of a graceful shutdown, the data is migrated to remaining active members.
-In version 5.5, there is no automatic data restoration in the event of an unexpected member loss.
+In the event of member crash, if backups were configured, they are used to restore the data.
+If no backups were configured, data can be lost.
+
+NOTE: You can disable backups to speed up data ingestion into Vector Collection and reduce memory usage
+in development and test environments which do not require fault tolerance.
+You do not have to do it in case of single-member cluster (for example embedded dev cluster) because in such case
+backups are not used.
```

Comment:
```suggestion
With a single-member cluster (for example, an embedded dev cluster), you do not need to disable backups because these are not used in this case.
```